### PR TITLE
Finishing Mobile Version.

### PR DESCRIPTION
**Pull request by Ashnaali3255**
**Description**
For the second milestone in building your portfolio website, you will create the mobile website section where you will list your portfolio projects.
**General requirements**
Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config):
Linter should be set up correctly for this repository.
The linter result should be green.
There should be no custom changes to the linter config.
Make sure that you used correctly [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md):
README file should be descriptive.
Commit messages should be meaningful.
Pull Request should have a descriptive title telling you what changes are inside.
Pull Request should have a short summary describing introduced changes in more detail.
Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md):
The branch should describe a feature or action (e.g. 'created_sidebar' or 'added_comments_section').
Pull Request is made from the 'feature branch' into the main branch.
HTML/CSS requirements
Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
Project requirements
IMPORTANT NOTE: For each Microverse project, you need to create a separate pull request. It is the first time you need to work with multiple pull requests in your one repository. [Read how to do it most efficiently](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/multiple_pull_requests.md).

Now you will build the middle section of the mobile website. We included a screenshot of Template 1 below for reference, but you should stick to the template that you chose. This sections of the mobile website should include:

"Works" section
Toolbar

the "About myself" section (placed after the projects and before the contact form):

Make sure the content of the "About myself" section is aligned with flexbox.

In order to lay out the elements in the "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

Please use ul and li for categories in the "Work" section. Html, CSS, and Javascript should be list items.

Please use ul and li for categories in the "About myself" section. Languages, Frameworks, and Skills should be listed items.

You don't need to implement the detail modal window that requires JavaScript.

You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma]

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,6 +34,112 @@ <h3>LET'S CONNECT</h3>
...
+        <li><img src="./Counter.jpg" alt="counter">2023</li>
+      </ul>
+    <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="list-flex">
+    <li class="language">html</li>
+    <li class="language">css</li>
+    <li class="language">javascript</li>
+    </ul>
+    <button><strong>See Project</strong></button>
+    </div>
```
Comment (Felix45) [inline index.html:91]: - [ Required ] Great job :clap: working on the `projects` section it looks good, just one observation that i think will improve your project. Please consider reducing the `margin-top` and `margin-top` properties between successive project cards, currently, it's bigger than what is provided on the `Figma template`

## Figma template


## Your design

File: index.html [hunk 2]
```diff
@@ -34,6 +34,112 @@ <h3>LET'S CONNECT</h3>
...
+          hestiate to contact me.</p>
+          <h3>LET'S CONNECT</h3>
+          <ul class="links link2">
+            <li><a title="twitter" href="https://twitter.com/Ashna_Ali1"><img src="./Twitter.jpg" alt="twitter"></a></li>
+            <li><a title="linkedIn" href="https://www.linkedin.com/in/ashna-ali-342151255/"><img src="./linkedin.jpeg" alt="linkedin"></a></li>
+            <li><a title="Medium" href="https://medium.com/Ashna_Ali1"><img src="./Medium.jpg" alt="medium"></a></li>
+           <li><a title="github" href="https://github.com/Ashnaali3255/Portfolio/tree/Mobile-Branch"><img src="./Github.jpg" alt="github"></a></li>
+           <li><a title="Angelist" href="https://angelist.com/Ashna_Ali1"><img src="./Angel.jpg" alt="angelist"></a></li>
+          </ul>
+          <button class="icon">Get My Resume</button>
```
Comment (Felix45) [inline index.html:126]: - [ Required ] Please consider adding button styling for the different button states `Active`, `Hover` and `Pressed` .  Please refer to this color palate on the Figma design for the exact color to use for different states.

## Figma template


```CSS
  button:hover {
   color: #fff;
   background-color: #6070ff;
}
  button:active {
    color: #fff;
    background-color: #2230d2;
}
```

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @Ashnaali3255 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [ ] All linter checks are setup and passing with green checks :heavy_check_mark: 
- [ ] GitHub flow is correctly used :heavy_check_mark: 
- [ ] Descriptive Pull Request title and summary :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -1,132 +1,323 @@
...
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  margin: 0 24px;
+}
+
+.projects {
+  margin: 0 24px;
+  padding: 20px 0;
```
Comment (franclobo) [inline style.css]: - [ ] Please 🙏🏽 use `padding: 0;` to reduce the space between the projects as requested by the previous reviewer to be similar to the Figma template.

File: style.css [hunk 4]
```diff
@@ -1,132 +1,323 @@
...
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr;
+}
+
+.project {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin-top: 114px;
```
Comment (franclobo) [inline style.css]: - [ ] Please 🙏🏽 use `margin-top: 88px;` to reduce the space between the projects as requested by the previous reviewer to be similar to the Figma template.

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Ashnaali3255, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- Professional README file ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (Ashnaali3255) [reply style.css]: @franclobo  Now I fix it please check it .

File: README.md [hunk 5]
```diff
@@ -21,7 +21,7 @@
 <!-- PROJECT DESCRIPTION -->
 
 # 📖 [Setup and Mobile First] <a name="about-project"></a>
-Create a HomePage for Portfolio, which can help to easy explorate profiles for Mobile Version.
+Create a Portfolio, which can help to easy explorate profiles for Mobile Version.
 
 ## 🛠 Built With <a name="built-with">
 #Tech Stack
```
Comment (Qoosim) [inline README.md:27]: - [OPTIONAL] Always try to add one or more future features under the `Future Features` heading to your projects. Kindly do that to this project and remember to not let it miss in your future projects.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @Ashnaali3255, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-   No linter errors. :heavy_check_mark: 
-   Good README file. :heavy_check_mark:
-   Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**